### ISSUE TITLE
Getting Started Code Only Returns Zeros

### ISSUE THREAD

**Issue by alexdriedger** [TriadSemi/triad_openvr#24]
After running the getting started code, the x, y, z coordinates in data are all 0s

```python
v = vr.triad_openvr()
data = v.devices["controller_1"].sample(1000,250)
print(data.x)
print(data.y)
print(data.z)
```

**Comment by ryan1288** [comment]
`v.devices["controller_1"].get_pose_euler()` also returns `None`, which causes the getting started code to crash

**Comment by dmitrySorokin** [comment]
Hi @ryan1288 have you solved this issue?

**Comment by tiaoyuedexin** [comment]
@dmitrySorokin ，I have the same problem，Have you solved it？

**Comment by dmitrySorokin** [comment]
@tiaoyuedexin, yes. My problem was that I placed the base station too close to the tracker. The distance should be ~ 1.5 - 2 meters.

**Comment by tiaoyuedexin** [comment]
@dmitrySorokin OK..so is my problem.thanks a lot
